src/xo-app/backup/new/index.js

```diff
@@ -118,7 +118,7 @@ const COMMON_SCHEMA = {
     },
     _reportWhen: {
       default: 'failure',
-      enum: ['never', 'always', 'failure'], // FIXME: can't translate
+      enum: ['never', 'always', 'failure or skipped'], // FIXME: can't translate
```

Comment:
IMHO it's not a good idea to change this.

Feel free to use `enumNames` to change what the user sees.